server/utils.go

```diff
@@ -83,3 +83,15 @@ func closeBody(r *http.Response) {
 		r.Body.Close()
 	}
 }
+
+func isValidAuthConfig(configuration *configuration) (bool, error) {
```

Comment:
I think this should just go in `configuration.go` -- IMO, `utils.go` should be for functions that are useful to multiple files.

Reply:
Agreed, moved to `configuration.go`